docs-gb/kubernetes/hpa-rps-autoscaling.md

```diff
@@ -246,29 +276,111 @@ spec:
         averageValue: 3
 ```
 
-In the two HPA manifests above, the scaling metric is exactly the same, and uses the exact same parameters: this is to ensure that both the Models and the Servers are scaled up/down at approximately the same time. Small variations in the scale-up time are expected because each HPA samples the metrics independently, at regular intervals. If a Model gets scaled up slightly before its corresponding Server, the model is currently marked with the condition ModelReady "Status: False" with a "ScheduleFailed" message until new Server replicas become available. However, the existing replicas of that model remain available and will continue to serve inference load.
+In the two HPA manifests above, the scaling metric is exactly the same, and uses the exact same
+parameters: this is to ensure that both the Models and the Servers are scaled up/down at
+approximately the same time. Small variations in the scale-up time are expected because each HPA
+samples the metrics independently, at regular intervals.
+
+{% hint style="info" %}
+If a Model gets scaled up slightly before its corresponding Server, the model is currently
+marked with the condition ModelReady "Status: False" with a "ScheduleFailed" message until new
+Server replicas become available. However, the existing replicas of that model remain available
+and will continue to serve inference load.
+{% endhint %}
+
+In order to ensure similar scaling behaviour between Models and Servers, the number of
+`minReplicas` and `maxReplicas`, as well as any other configured scaling policies should be kept
+in sync across the HPA for the model and the server.
+
+#### Details on custom metrics of type Object
+
+The HPA manifests above use metrics of type "Object" that fetch the data used in scaling
+decisions by querying k8s metrics associated with a particular k8s object.  The endpoints that
+HPA uses for fetching those metrics are the same ones that we tested in the previous section
+using `kubectl get --raw ...`. Because we have configured the Prometheus Adapter to expose those
+k8s metrics based on queries to Prometheus, a mapping exists between the information contained
+in the HPA Object metric definition and the actual query that is executed against Prometheus.
+This section aims to give more details on how this mapping works.
+
+In our example, the `metric.name:infer_rps` gets mapped to the `seldon_model_infer_total` metric
+on the prometheus side, based on the configuration in the `name` section of the Prometheus
+Adapter ConfigMap. The prometheus metric name is then used to fill in the `<<.Series>>` template
+in the query (`metricsQuery` in the same ConfigMap).
+
+Then, the information provided in the `describedObject` is used within the Prometheus query to
+select the right aggregations of the metric. For the RPS metric we have used to scale the Model
```

Comment:
```suggestion
select the right aggregations of the metric. For the RPS metric scale the Model
```

Reply:
A different change is needed here, as now the sentence no longer makes sense. Will rephrase to "For the RPS metric used to scale the Model"